internal/configs/virtualserver_test.go

```diff
@@ -4927,7 +4927,7 @@ func TestGenerateSSLConfig(t *testing.T) {
 				RejectHandshake: true,
 			},
 			expectedWarnings: Warnings{
-				nil: []string{"TLS secret secret is invalid: secret doesn't exist"},
+				nil: []string{"TLS secret is invalid: secret doesn't exist"},
```

Comment:
Changing this (and the following one) will make the test fail. The second `secret` here is just the name specified on line `4916`.